src/game.cpp

```diff
@@ -8839,6 +8854,12 @@ bool game::walk_move( const tripoint &dest_loc )
     if( m.impassable( dest_loc ) && !pushing && !shifting_furniture ) {
         return false;
     }
+    vehicle *veh = veh_pointer_or_null( m.veh_at( dest_loc ) );
+    if( veh ) {
+        if( !veh->handle_potential_theft( &g->u ) ) {
```

Comment:
```suggestion
        if( !veh->handle_potential_theft( &u ) ) {
```
We're in `game.cpp` anyway.